CMakeLists.txt

```diff
@@ -111,8 +118,12 @@ else()
 	set(CMAKE_INSTALL_INCLUDEDIR "include")
 endif()
 
-set(LIBDIR "${CMAKE_INSTALL_LIBDIR}" CACHE PATH "Installed library directory")
-mark_as_advanced(LIBDIR)
+if(NOT LIBMGBA_ONLY)
+	set(LIBDIR "${CMAKE_INSTALL_LIBDIR}" CACHE PATH "Installed library directory")
+	mark_as_advanced(LIBDIR)
+elseif(NOT DEFINED LIBDIR)
+	set(LIBDIR "${CMAKE_INSTALL_LIBDIR}")
```

Comment:
It might be better to just remove the use of `LIBDIR` entirely. I'm pretty sure it's legacy.

Reply:
The goal of this PR is to remove side effects of libmgba without altering its behavior when building alone, I think it's fair to consider that to be out of scope
I could remove it of that's prefered, should that also include LIBRETRO_DIR and OE_DIR ?